Gemfile

```diff
@@ -80,3 +80,12 @@ end
 
 gem "govuk_notify_rails", "~> 2.2", ">= 2.2.0"
 
+group :ui_auto do
+  gem 'capybara'
+  gem 'pry'
+  gem 'require_all'
```

Comment:
We can do this with a one-liner and avoid adding another dependency

Reply:
removed. now using local require_all method in ui_spec_helper file